### PR TITLE
Update "Publication Policies"

### PR DESCRIPTION
**Pull request by aguilerapy**
Change the name from "Publication Policies" to "Publication Policies: Data User Guide".

Based on CRM issue 5167.

### CODE REVIEW

**Comment by jpmckinney** [comment]
@aguilerapy This will make the menu item in the left-hand sidebar very long (maybe longer than the sidebar): https://standard.open-contracting.org/latest/en/implementation/publication_policy/

Also, I think we'd want to edit some of the text on the page to more consistently use 'data user guide' instead of (or along with) publication polic*.

**Comment by aguilerapy** [comment]
2019-10-18-140152_1121x670_scrot

@jpmckinney The left-hand sidebar don't change, the text "Guide" ran to the second line. 

2019-10-18-140210_998x678_scrot

We can also leave "Data User Guide" as subtitle in publication_policy.md what do you think?

**Comment by jpmckinney** [comment]
Yes, my concern is that it goes onto two lines, whereas we try to keep each menu item on one line.

The subtitle will unfortunately create an extra layer of hierarchy, which isn't something we want.

I think we should focus on editing the text to make it clearer that this policy should function as a data user guide (we can bold those words). 

We should work to align the content from the newer publication policy guidance ([EN](https://www.open-contracting.org/resources/ocds-1-1-publication-policy-template/), [ES](https://www.open-contracting.org/resources/ocds-1-1-plantilla-de-politica-de-publicacion-es/)) with this page.

That would be a bigger project, but it will have more impact than adding a few words to this page.

**Comment by aguilerapy** [comment]
Okay, I will work on that

File: standard/docs/en/implementation/publication_policy.md [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+4. Any exclusions from the data - such as information which is not disclosed for reasons of privacy and security. This can describe the fields which are excluded for a selection of records, and/or the policy applied to decide if any information is withheld from a public OCDS release. Highlighting how their legal frameworks and concepts map onto their OCDS publication covering.
+5. Any custom codelists used - or custom codes added to existing codelists.
+6. Who is responsible for providing this data - including appropriate contact details and locations of where to offer feedback on the data or seeking clarification on its use.
+7. Details of the license under which the data is published.
+8. Any relevant additional documentation that the reader should be aware of when using a publishers OCDS dataset.
+9. Future development plans - brief notes on the future plans of the publisher to improve their data.
+10. The limits of the publishers legal responsibilities for the data.
```
Comment (aguilerapy) [inline standard/docs/en/implementation/publication_policy.md]: Are the summary sections that are in the template drive document.

**Comment by jpmckinney** [comment]
@aguilerapy Let me know when there is a new version ready for review (no rush).

**Comment by aguilerapy** [comment]
@jpmckinney free to review!

**Comment by jpmckinney** [comment]
@LindseyAM to review

**Comment by LindseyAM** [comment]
Thanks both! I've pulled together some (even shorter) text in this google doc.

My understanding it will fall under the Publish subheading beneath the Guidance Heading in 1.1.5

James, feel free to make any edits and incorporate into documentation

https://docs.google.com/document/d/1cArKYbzYpfVt7WuIPTZz2Ot2JLahhESgc_q6SFeGSnA/edit#

**Comment by jpmckinney** [comment]
After merging, check if #759 is closed.

**Comment by jpmckinney** [comment]
The text from the linked document will likely be merged as part of #986. Once that's done, I can review the changes here, to see if we missed anything.

**Comment by jpmckinney** [comment]
Superseded by #986.

**Pull request closed without merging**